tests/unit/test_processors.py

```diff
@@ -14,8 +14,35 @@
 from circuit_maintenance_parser.parser import Parser
 
 
-PARSED_DATA = [{"a": "b"}, {"c": "d"}]
-EXTENDED_DATA = {"z": "x"}
+PARSED_DATA = [
```

Comment:
I would keep the `fake` data style, to make this completely independent from th eactual values from the parsers

Reply:
@carbonarok remember to address this too